src/getters/getBlockExplorerUrl.ts

```diff
@@ -1,8 +1,8 @@
-import getMainChainId from '~/getters/getMainChainId'
 import getCoreConfig from './getCoreConfig'
+import { defaultChainConfig } from './getChainConfig'
 
 export function getBlockExplorerUrl(chainId?: number): string {
-    const actualChainId = chainId ?? getMainChainId()
+    const actualChainId = chainId ?? defaultChainConfig.id
```

Comment:
`getBlockExplorerUrl` is used once and without the `chainId`. Is this ok?

Maybe let's make `chainId` mandatory. Defaults tend to cause head scratches.